geoportal/vars.yaml

```diff
@@ -868,6 +868,9 @@ vars:
         - https://127.0.0.1:3000
         - https://127.0.0.1:3001
         - '*'
+        # GeoGirafe
+        - 'https://app.localhost:8080'
```

Comment:
What is this URL?

Reply:
It's created by docker compose?

Reply:
No, it is created with vitejs. Why ?

Reply:
I'm relay surprise to see the `app.` prefix!